libsql_client/client.py

```diff
@@ -97,4 +101,6 @@ def _normalize_value(in_value: InValue) -> Value:
         return int(in_value.timestamp() * 1000)
     elif isinstance(in_value, bool):
         return int(in_value)
-    return in_value
+    elif isinstance(in_value, (str, float)) or in_value is None:
```

Comment:
I think we missed `int` in the list, that's why the CI failed: `(str, int, float)`